src/http-api.ts

```diff
@@ -376,19 +376,46 @@ export class HttpApi {
 
   #setupWeb5ConnectServerRoutes(): void {
     /**
-     * Endpoint that the connecting App pushes the Pushed Authorization Request Object to start the Web5 Connect flow.
-     */
+    * Endpoint allows a Client app (RP) to submit an Authorization Request.
+    * The Authorization Request is stored on the server, and a unique `request_uri` is returned to the Client app.
+    * The Client app can then provide this `request_uri` to the Provider app (wallet).
+    * The Provider app uses the `request_uri` to retrieve the stored Authorization Request.
+    */
     this.#api.post('/connect/par', async (req, res) => {
       log.info('Storing Pushed Authorization Request (PAR) request...');
 
+    // TODO: Add validation for request too large HTTP 413: https://github.com/TBD54566975/dwn-server/issues/146
+    // TODO: Add validation for too many requests HTTP 429: https://github.com/TBD54566975/dwn-server/issues/147
+
+      if (!req.body.request) {
+        return res.status(400).json({
+          ok: false,
+          status: {
+            code: 400,
+            message: "Bad Request: Missing 'request' parameter",
+          },
+        });
+      }
+
+      // Validate that `request_uri` was NOT provided
+      if (req.body?.request?.request_uri) {
+        return res.status(400).json({
+          ok: false,
+          status: {
+            code: 400,
+            message: "Bad Request: 'request_uri' parameter is not allowed in PAR",
+          },
+        });
+      }
```

Comment:
Sanity: are the new code covered by tests? If not, please makes sure we have 100% new code coverage.

Reply:
> please makes sure we have 100% new code coverage

why? these are harmless validations not on the golden path.

Reply:
Hey @shamilovtim, are you suggesting that we should only test golden paths? Harmless validations still should be tested no? It'll take probably take longer time arguing about it than just adding tests to it when it is easy to do so in JS/TS.

In the spirit of this week's deadline, you are good to resolve this and merge, but I've filed an issue #149 and assigned to you as a fast-follow.

Reply:
> are you suggesting that we should only test golden paths? Harmless validations still should be tested no?

@thehenrytsai  Yes, I'm saying that all code does not need to be tested. I don't think that's a controversial statement. 100% coverage is actually a far more controversial thing to suggest.